<!-- xtion_camera_ir.launch -->
<launch>

	<!-- chessboard size as NxM, counting interior corners -->
	<arg name="size" value="8x6" />

	<!-- chessboard square size in meters -->
	<arg name="square" value="0.024" />

	<!-- By default, calibrations are stored to file://${ROS_HOME}/camera_info/${NAME}.yaml,
		where ${NAME} is of the form "[rgb|depth]_[serial#]", e.g. "depth_B00367707227042B".
		See camera_info_manager docs for calibration URL details. -->
	<arg name="depth_camera_info_url" default="file://$(find camera)/info/camera_info/depth_xtion_camera.yaml" />

	<include file="$(find openni2_launch)/launch/openni2.launch" >
		<arg name="depth_camera_info_url"	value="$(arg depth_camera_info_url)" />
	</include>

	<node name="contrast_augmenter" pkg="camera" type="contrast_augmenter" />

	<node name="calibrator" pkg="camera_calibration" type="cameracalibrator.py"
		args="--size $(arg size) --square $(arg square)"
		output="screen" clear_params="true">
		<remap from="image"		to="/camera/ir_augmented/image" />
		<remap from="camera"	to="/camera/ir" />
	</node>

</launch>
